scripts/Client.js

```diff
@@ -39,66 +40,67 @@ function getRow(row) {
     props.ContactOption = values[0][5];
     props.CalleeNote = values[0][6];
     props.ContactDate = "";
+    //console.log((values[0]));
 //*/
     
-  }).getRow(SPREADSHEET_ID, row);
+  }).getRow(SPREADSHEET_ID, row, campaign);
 }
 
 
 
-
-function getRange(row) {
+function getRange(row) { 
```

Comment:
you'd initialize the global CAMPAIGN here as this is the onClick() for 'get your data'